client/v2_2/save_.py

```diff
@@ -22,6 +22,7 @@
 import io
 import json
 import os
+import sys
```

Comment:
Use logging instead, you can check fast_flatten or fast_pusher for reference.

Reply:
See my commit message for explanation why I don't want to use `logging`.
Also, I would like the print to go to stderr, not stdout, because other bazel build status messages also go to stderr:
```
Loading: 
Loading: 0 packages loaded
    currently loading: foo
Analyzing: target //foo:bar (45 packages loaded)
...
```
I think the pull status progresses should appear together with these messages in stderr like so:
```
Loading: 
Loading: 0 packages loaded
    currently loading: foo
Analyzing: target //foo:bar (45 packages loaded)
Downloading from gcr.io/tensorflow/tensorflow:latest (1/12)
Downloading from gcr.io/tensorflow/tensorflow:latest (2/12)
Downloading from gcr.io/tensorflow/tensorflow:latest (3/12)
...
```
If the output goes to stdout, then these status messages would clutter run_log.txt in the following invocation:
```
bazel run //foo:bar 2>build_log.txt >run_log.txt
```